ouroboros-consensus/src/consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Adversarial.hs

```diff
@@ -73,29 +73,29 @@ data AdversarialViolation hon adv =
     BadRace !(RaceViolation hon adv)
   deriving (Eq, Read, Show)
 
--- | Check the chain matches the given 'AdversarialRecipe', especially the Extended Praos Race Assumption
+-- | Check the chain matches the given 'AdversarialRecipe', especially that it
+-- satisfies Adversarial Chains in Stability Windows and Length of Competing
+-- Chains.
 --
--- Definition of a /Praos Race Window/. Such a window includes exactly 'Delta'
--- slots after the @k+1@st honest block in the window.
+-- Definition of a /Praos Race Window/ of a chain. It is an interval of slots
+-- that contains at least @k+1@ blocks of the chain and exactly 'Delta' slots
+-- after the @k+1@st block.
 --
--- Definition of the /Praos Race Assumption/. We assume every adversarial chain
--- contains at most @k@ blocks in the Praos Race Window anchored at the
```

Comment:
Minor: "anchored" is a well-established term in the rest of the Consensus code base and documentation. I generally suggest not replacing uses of it.

Reply:
Do you mean window anchoring? It took me a bit to find the definition of that. But I don't feel strong. I'll reintroduce it.

Reply:
I just realized this occurrence of anchored is preserved in the current formulation of PRA as LoCCA. Please, let me know if any action is needed to address this comment.